common/src/main/java/com/wynntils/features/user/AddCommandExpansionFeature.java

```diff
@@ -62,6 +62,7 @@ private void addArgumentlessCommandNodes(RootCommandNode root) {
         root.addChild(literal("hub").build());
         root.addChild(literal("itemlock").build());
         root.addChild(literal("kill").build());
+        root.addChild(literal("hub").build());
```

Comment:
Nice catch, but don't mix fixes in PRs next time, please.

Reply:
Also, was it a catch? `hub` was already added, in it's alphabetic place.